x-pack/plugins/maps/public/actions/store_actions.js

```diff
@@ -359,6 +360,9 @@ export function clearMouseCoordinates() {
   return { type: CLEAR_MOUSE_COORDINATES };
 }
 
+export function setScrollZoom(scrollZoom) {
```

Comment:
up2you, but we're never turning this on. would change this to `disableScrollZoom` and remove param.